collections/_article/d3d12-shaders-with-a-twist.md

```diff
@@ -0,0 +1,112 @@
+---
+title: "Direct3D 12: Shaders with a twist"
+excerpt: "Godot is about to get a Direct3D 12 rendering driver. This is a discussion on the exotic approaches taken to face some of the challenges posed in the area of shaders."
+categories: ["progress-report"]
+author: Pedro J. Estébanez
+image: /storage/blog/covers/d3d12-shaders-with-a-twist.jpg.jpg
+date: 2023-04-12 20:00:00
+---
+
+This is Pedro, a.k.a. [RandomShaper](https://twitter.com/RandomPedroJ) in the Godot community.
+
+You may already know what I have been implementing a **Direct3D 12 rendering driver** for Godot. Since [the announcement](https://w4games.com/2022/08/12/w4-games-donates-direct3d-12-support-to-godot/) that W4 Games donated my time working on that to the project, the code has received an amount of maintenance. In fact, [the pull request](https://github.com/godotengine/godot/pull/70315) hasn't still been merged since there has been a continuous flow of improvements.
```

Comment:
```suggestion
You may already know that I have been implementing a **Direct3D 12 rendering driver** for Godot. Since [the announcement](https://w4games.com/2022/08/12/w4-games-donates-direct3d-12-support-to-godot/) that W4 Games donated my time working on that to the project, the code has received a lot of maintenance. In fact, [the pull request](https://github.com/godotengine/godot/pull/70315) hasn't still been merged since there has been a continuous flow of improvements.
```